scripts/reports/new_data.Rmd

```diff
@@ -51,14 +54,15 @@ df %>%
     y = "Total Confirmed Flu Admissions"
   ) +
   theme(axis.text.x = element_text(angle = 90, hjust = 1))
+ggplotly(p, tooltip = "text", height = 800, width = 1000)
```

Comment:
is this how we get the hovertext to actually work? the lack of that has meant needing to do some random `tar_reads`

Reply:
i didnt actually try to fix that issue, this fix is to make the HTML self-contained rather than producing a folder with PNGs